conformance/src/test/scala/za/co/absa/enceladus/samples/TradeConformance.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.samples
+
+import za.co.absa.enceladus.model.conformanceRule._
+import za.co.absa.enceladus.model.{Dataset, DefaultValue, MappingTable}
+import za.co.absa.enceladus.utils.error.ErrorMessage
+import za.co.absa.enceladus.utils.error.Mapping
+
+object TradeConformance {
+  val countryMT = MappingTable(name = "country", version = 0, hdfsPath = "src/test/testData/country",
+    schemaName = "country", schemaVersion = 0)
+
+  val countryRule = MappingConformanceRule(order = 0, mappingTable = "country", controlCheckpoint = true,
+    mappingTableVersion = 0, attributeMappings = Map("country_code" -> "legs.conditions.country" ),
+    targetAttribute = "country_name", outputColumn = "legs.conditions.conformed_country")
+
+  val litRule = LiteralConformanceRule(order = 3, outputColumn = "MyLiteral", controlCheckpoint = true, value = "abcdef")
+
+  val upperRule = UppercaseConformanceRule(order = 4, inputColumn = "MyLiteral", controlCheckpoint = false,
+    outputColumn = "MyUpperLiteral")
+
+  val lit2Rule = LiteralConformanceRule(order = 5, outputColumn = "ToBeDropped", controlCheckpoint = true,
+    value = "123456")
+
+  val dropRule = DropConformanceRule(order = 6, outputColumn = "ToBeDropped", controlCheckpoint = false)
+
+  val concatRule = ConcatenationConformanceRule(order = 7, outputColumn = "Concatenated", controlCheckpoint = true,
+    Seq("MyLiteral", "MyUpperLiteral"))
+
+  val singleColRule = SingleColumnConformanceRule(order = 9, outputColumn = "legs.ConformedLegId",
+    controlCheckpoint = true, inputColumn = "legs.legid", inputColumnAlias = "legId")
+
+  val tradeDS = Dataset(name = "Trade Conformance", version = 1, hdfsPath = "src/test/testData/trade",
+    hdfsPublishPath = "testData/conformedTrade",
+      schemaName = "Employee", schemaVersion = 0,
+      conformance = List(countryRule, litRule, upperRule, lit2Rule, dropRule, concatRule, singleColRule) )
+
+  // The original sample used to create the parquet file
+  // import spark.implicits._
+  // val dfA = spark.read.json(sample.toDS)
+  // dfA.repartition(1).write.parquet("/Users/abri036/sources/enceladus/conformance/src/test/testData/trade/2017/11/01/")
```

Comment:
I don't think this comment belongs here, with an absolute path.

Reply:
oops